src/components/SearchSelect.vue

```diff
@@ -0,0 +1,96 @@
+<template>
+  <app-select
+    v-model="selectedEntity"
+    size="sm"
+    class="search-select"
+    :options="entities"
+    track-by="name"
+    label="name"
+    :hide-selected="true"
+    :searchable="false"/>
+</template>
+
+<script setup>
+import { ref } from 'vue'
+import { useVModel } from '@vueuse/core'
+import { useRoute } from '#app'
+import AppSelect from '@/components/AppSelect'
+
+const route = useRoute()
+
+const emit = defineEmits(['update:modelValue'])
+
+const props = defineProps({
+  modelValue: {
+    type: Object,
+    default: null,
+  },
+})
+
+onMounted(() => {
+  preselectedEntityName.value = route.path.split('/')[1] || ''
+  selectedEntity.value = preselectedEntity.value
+})
+
+const selectedEntity = useVModel(props, 'modelValue', emit)
+
+const preselectedEntity = computed(() => {
+  return entities.value.find(entity => entity.url === preselectedEntityName.value)
+})
+
+const preselectedEntityName = ref(null)
```

Comment:
The dropdown is not in scope for v1, as we discussed but I see it's implemented 🤔

Reply:
yeah and this is exactly why I have been asking in the issue. it does not make sense to filter the search for all possible entities.

Reply:
I don't even think it makes sense at all to (pre-)filter searches. a search should be convenient and if there are different result entities, it should be reflected in the search result instead

Reply:
There was a misunderstanding in the description and order in which task should come. It will be resolved diffrently